ParentWorkspace/SpringModule/src/main/java/com/miniproject/model/Cart.java

```diff
@@ -0,0 +1,67 @@
+package com.miniproject.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+/**
+ * @author Mohaneswaran
+ *
+ *         Maps with user and item table with many to one cardinality.
+ */
+
+@Entity
+@Table(name = "CART")
+
+public class Cart {
+	
+	@OneToOne
+	@JoinColumn(name = "USERID", nullable = false)
+	private User userId;
+	
+	@OneToMany
+	@JoinColumn(name = "ITEMID", nullable = false)
+	private Item itemId;
+	
+	@Column(name = "QUANTITY")
+	private Integer quantity;
+	
+	
+
+	public Cart(User userId, Item itemId, Integer quantity) {
+		super();
```

Comment:
its not required to call super constructor explicitly until the super class is user defined.
In this case super class would be Object and super() is automatically called by java